lib/Conversion/HandshakeToFIRRTL/HandshakeToFIRRTL.cpp

```diff
@@ -76,6 +77,16 @@ static FIRRTLType getFIRRTLType(Type type) {
         unsigned width = indexType.kInternalStorageBitWidth;
         return UIntType::get(context, width);
       })
+      .Case<TupleType>([&](TupleType tupleType) -> FIRRTLType {
+        using BundleElement = BundleType::BundleElement;
+        llvm::SmallVector<BundleElement, 2> elements;
```

Comment:
nit: unless you're reasonably sure 2 is a good number, you can leave it off and SmallVector will pick a size that fits neatly on the stack: https://llvm.org/docs/ProgrammersManual.html#llvm-adt-smallvector-h